hubblestack/files/hubblestack_nova/grep.py

```diff
@@ -121,41 +121,41 @@ def audit(data_list, tags, debug=False, **kwargs):
                 if grep_ret:
                     found = True
                     failure_reason = "Found the blacklisted string '{0}' in file '{1}'." \
-                                    " The file should not contain any string like '{2}'" \
-                                    .format(grep_ret,
```

Comment:
Do you use some tool or plugin to auto-indent?
If yes, let me know about it.

Reply:
Vim does it for me on the third line. But not the second. I just am OCD about that kind of stuff.